Sources/ParseSwift/Objects/ParseObject.swift

```diff
@@ -54,42 +59,166 @@ public extension Sequence where Element: ParseObject {
 
     /**
      Saves a collection of objects *synchronously* all at once and throws an error if necessary.
-
-     - parameter options: A set of options used to save objects. Defaults to an empty set.
+     - parameter batchLimit: The amount of objects to send in each batch. If the items to be batched
```

Comment:
I think this is slightly better...
```suggestion
     - parameter batchLimit: The maximum number of objects to send in each batch. If the items to be batched
```

Reply:
Agreed